cmd/vclusterctl/cmd/completion.go

```diff
@@ -0,0 +1,92 @@
+package cmd
+
+import (
+	"errors"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+const compDesc = `
+#######################################################
+################### vcluster complete #################
+#######################################################
+Generates completion scripts for various shells
+
+Example:
+vcluster completion bash
+vcluster completion zsh 
+#######################################################
+`
+
+const bashCompDesc = `
+#######################################################
+################### vcluster complete bash ############
```

Comment:
Please rename to vcluster completion bash as the sub command is called completion as well